tests/test_examples.py

```diff
@@ -19,7 +20,13 @@ def setup():
 
 
 def test_mnist():
-    mnist_test()
+    filename = 'mnist.pkl'
```

Comment:
Would be nice! In theory we should have done everything without using the file system altogether, but sometimes using a real file is just a way simpler.